src/Rest/Handler/Mappings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Rest\Handler;
+
+use MediaWiki\Rest\Response;
+use MediaWiki\Rest\ResponseException;
+use MediaWiki\Rest\SimpleHandler;
+use Wikimedia\ParamValidator\ParamValidator;
+
+class Mappings extends SimpleHandler {
+
+	private ?string $method = null;
+
+	public static function factory(): self {
+		$self = new self();
+
+		return $self;
+	}
+
+	public function setMethodFromRequest(): void {
+		$method = $this->getRequest()->getMethod();
+
+		$this->method = strtoupper( $method );
+	}
```

Comment:
Can't you just do `getMethod(): string`? Avoiding calling that twice is not worth the complexity of an additional field.

Plus you could use a match statement to avoid calling it twice: https://www.php.net/manual/en/control-structures.match.php